contracts/PriceOracle/PriceOracle.sol

```diff
@@ -68,13 +68,20 @@ contract PriceOracle is Ownable2Step {
     /// @notice The number of digits the price is scaled to before adjusted by the base units
     uint8 internal constant PRICE_SCALE = 36;
 
+    /// @notice The minimum number of digits of precision to preserve when formatting by restricting the underlying asset decimals config
+    /// @dev Minimum precision is set to 6 to match the Compound UniswapAnchoredView behavior
+    uint8 internal constant MIN_PRECISION = 6;
```

Comment:
Why did we choose 6?

Reply:
It was chosen to match the existing behavior of the UAV shown [here](https://github.com/smartcontractkit/open-oracle/blob/48c123d33103c4790d1b140f778ac7d2a5d1c6ad/contracts/Uniswap/UniswapAnchoredView.sol#L214)